microsoft-edge/devtools-guide-chromium/shortcuts/index.md

```diff
@@ -48,23 +48,23 @@ The following keyboard shortcuts are available in most DevTools panels.
-| Toggle [Device emulation](../device-mode/index.md) | `Ctrl`+`Shift`+`M` | `Command`+`Shift`+`M` |
-| Toggle **Inspect Element Mode** | `Ctrl`+`Shift`+`C` | `Command`+`Shift`+`C` |
-| Open the [Command Menu](../command-menu/index.md) | `Ctrl`+`Shift`+`P` | `Command`+`Shift`+`P` |
-| Toggle the [Drawer](../customize/index.md#drawer) | `Escape` | `Escape` |
-| Normal refresh | `F5` or `Ctrl`+`R` | `Command`+`R` |
-| Hard refresh | `Ctrl`+`F5` or `Ctrl`+`Shift`+`R` | `Command`+`Shift`+`R` |
-| Search for text within the current panel.  Not supported in the **Audits**, **Application**, and **Security** tools | `Ctrl`+`F` | `Command`+`F` |
-| Opens the **Search** tab in the [Drawer](../customize/index.md#drawer), which lets you search for text across all loaded resources | `Ctrl`+`Shift`+`F` | `Command`+`Option`+`F` |
-| Open a file in the **Sources** tool | `Ctrl`+`O` or `Ctrl`+`P` | `Command`+`O` or `Command`+`P` |
-| Zoom in | `Ctrl`+`Shift`+`+` | `Command`+`Shift`+`+` |
-| Zoom out | `Ctrl`+`-` | `Command`+`-` |
-| Restore default zoom level | `Ctrl`+`0` | `Command`+`0` |
-| Run snippet | Press `Ctrl`+`O` to open the [Command Menu](../command-menu/index.md), type `!` followed by the name of the script, then press `Enter` | Press `Command`+`O` to open the [Command Menu](../command-menu/index.md), type `!` followed by the name of the script, then press `Enter` |
+| Show **Settings** | **Shift+?** or **F1** | **Shift+?** or **Function+F1** |
+| Focus the next panel | **Ctrl+]** | **Command+]** |
+| Focus the previous panel | **Ctrl+[** | **Command+[** |
+| Switch back to whatever [docking position](../customize/placement.md) you last used.  If DevTools has been in the default position for the entire session, then this shortcut undocks DevTools into a separate window. | **Ctrl+Shift+D** | **Command+Shift+D** |
+| Toggle [Device emulation](../device-mode/index.md) | **Ctrl+Shift+M** | **Command+Shift+M** |
+| Toggle **Inspect Element Mode** | **Ctrl+Shift+C** | **Command+Shift+C** |
+| Open the [Command Menu](../command-menu/index.md) | **Ctrl+Shift+P** | **Command+Shift+P** |
+| Toggle the [Drawer](../customize/index.md#drawer) | **Esc** | **Esc** |
+| Normal refresh | **F5** or **Ctrl+R** | **Command+R** |
+| Hard refresh | **Ctrl+F5** or **Ctrl+Shift+R** | **Command+Shift+R** |
+| Search for text within the current panel.  Not supported in the **Audits**, **Application**, and **Security** tools | **Ctrl+F** | **Command+F** |
+| Opens the **Search** tab in the [Drawer](../customize/index.md#drawer), which lets you search for text across all loaded resources | **Ctrl+Shift+F** | **Command+Option+F** |
+| Open a file in the **Sources** tool | **Ctrl+O** or **Ctrl+P** | **Command+O** or **Command+P** |
+| Zoom in | **Ctrl+Shift**+**+** | **Command+Shift**+**+** |
```

Comment:
@dan-wesley  - special case, I un-bolded the connector + , pls see rendered result

Reply:
Correct, shift is not needed to zoom in or out.